src/command/schema/push.rs

```diff
@@ -0,0 +1,70 @@
+use crate::client::get_rover_client;
+use anyhow::Result;
+use rover_client::query::schema::push;
+use std::path::{Path, PathBuf};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct Push {
+    /// where to find the schema. .graphql, .json or uri
+    #[structopt(name = "SCHEMA_PATH", parse(from_os_str))]
```

Comment:
does `from_os_str` handle URIs? i'm wondering if perhaps we should make this a bit more clear with some clap/structopt features that allow you to express this a bit differently (<PathBuf | Uri>) in the usage help text might be nice